source/includes/_tipser-react-elements.md

```diff
@@ -62,7 +64,7 @@ ReactDOM.render(
     </TipserElementsProvider>, 
     document.getElementById('root'));
 ```
-Make sure your HTML document contains an element of id `root` (`<div id="root"/>`) so that React can mount the app to your HTML document, provide `history` object on which we can rely for client side routing, when not provided routing is based on `window.history`. Also check our [configuration](#configuration-options) options.
+Make sure your HTML document contains an element of id `root` (`<div id="root"/>`) so that React can mount the app to your HTML document, provide `history` object on which we can rely for client side routing, when not provided routing is based on `window.history` and import CSS files. Also check our [configuration](#configuration-options) options.
```

Comment:
```suggestion
Make sure:
- your HTML document contains an element of id `root` (`<div id="root"/>`) so that React can mount the app to your HTML document
- to provide `history` object on which we can rely for client side routing, when not provided routing is based on `window.history` 
- to import CSS files. 

Also check our [configuration](#configuration-options) options.
```